experiment/tracer/static/script.js

```diff
@@ -0,0 +1,273 @@
+'use strict';
+
+var logger = {
+  data: [],
+  message: '',
+  options: {
+    request: {
+      endpoint: 'https://' + window.location.hostname + '/trace?',
```

Comment:
the rest of the deck pages can be iterated on locally with [runlocal](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/deck/runlocal), it's a bit unfortunate that this one won't.

Reply:
We don't use /trace anymore.